test/lib/gobgp.py

```diff
@@ -275,12 +275,15 @@ def monitor():
         t.daemon = True
         t.start()
 
-    def _get_adj_rib(self, adj_type, peer, prefix='', rf='ipv4'):
+    def _get_adj_rib(self, adj_type, peer, prefix='', rf='ipv4', add_path_enabled=False):
         peer_addr = self.peer_name(peer)
         cmd = 'gobgp neighbor {0} adj-{1} {2} -a {3} -j'.format(peer_addr,
                                                                 adj_type,
                                                                 prefix, rf)
         output = self.local(cmd, capture=True)
+        if add_path_enabled:
+            return self._get_rib(json.loads(output))
+
```

Comment:
This is the issue with this test function:
`ret = [p[0] for p in json.loads(output).values()]`
It only returns the first path. We should change this, but it involves changing a lot of tests. So I have added a new parameter for this PR.